src/DotNetLightning.Core/Chain/KeysInterface.fs

```diff
@@ -89,7 +89,7 @@ type IKeysRepository =
     /// restarted with some stale data.
     abstract member GetChannelKeys: inbound:bool -> ChannelKeys
     /// Get a secret for constructing onion packet
-    abstract member GetSessionKey: unit -> Key
+    //abstract member GetSessionKey: unit -> Key
```

Comment:
commented code, rather just remove the line or explain (in a comment) why keep it there commented